cmd/minikube/cmd/config/addons_list.go

```diff
@@ -166,13 +166,19 @@ var printAddonsJSON = func(cc *config.ClusterConfig) {
 
 		addonBundle := assets.Addons[addonName]
 		enabled := addonBundle.IsEnabled(cc)
-
+	        maintainer := addonBundle.Maintainer
+		docs := addonBundle.Docs
 		addonsMap[addonName] = map[string]interface{}{
 			"Status":  stringFromStatus(enabled),
 			"Profile": cc.Name,
+			"maintainer": maintainer,
+			"docs":       docs,
```

Comment:
```suggestion
			"Maintainer": maintainer,
			"Docs":       docs,
```
For consistency sake we should keep the keys starting with a capital.